### ISSUE TITLE
[Fixed in GH-6052] Missing visual focus on the Continue button on checkout shipping address page

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#5039]
<img width="1153" alt="Screen Shot 2019-10-16 at 1 51 22 PM" src="https://user-images.githubusercontent.com/45409515/66944941-2c084200-f01c-11e9-82d2-106a76bbe9d0.png">

**Comment by KateChuen** [comment]
On all 3 browsers: 
- Still missing focus on the "Continue" button.
- The "Add New Address" and "Back to Cart" buttons have a grey focus instead of the blue one.

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).


![Screen Shot 2020-01-31 at 2 58 03 PM](https://user-images.githubusercontent.com/15113219/73570292-218e7e80-443a-11ea-91c6-0f9ce6ab693e.png)
![Screen Shot 2020-01-31 at 2 56 09 PM](https://user-images.githubusercontent.com/15113219/73570293-218e7e80-443a-11ea-963e-385f21c4793f.png)
![Screen Shot 2020-01-31 at 2 56 50 PM](https://user-images.githubusercontent.com/15113219/73570294-218e7e80-443a-11ea-8b65-493ddc43ccd2.png)

**Comment by KateChuen** [comment]
Tested on dev3
- Chrome ✅
- Firefox ✅
- Safari ✅
